WorkloadManagementSystem/Agent/StatesMonitoringAgent.py

```diff
@@ -55,7 +54,10 @@ def initialize( self ):
     self.monitoringDB = MonitoringDB()
 
     self.am_setOption( "PollingTime", 120 )
```

Comment:
well now it make no sense, but it is the report period. We have to tune later on this number...